Tests/scripts/validate_files_structure.py

```diff
@@ -682,6 +703,203 @@ def str2bool(v):
         raise argparse.ArgumentTypeError('Boolean value expected.')
 
 
+def is_text_file(file_path):
+    file_extension = os.path.splitext(file_path)[1]
+    text_file_types = set(['.yml', '.py', '.json', '.md', '.txt', '.sh', '.ini', '.eml', '', '.csv'])
+    if file_extension in text_file_types:
+            return True
+    return False
+
+
+def get_diff_text_files(files_string):
+    accepted_file_statuses = ['M', 'A']
+    all_files = files_string.split('\n')
+    text_files_list = set([])
```

Comment:
why are you converting a list to a set and not just creating a set in the first place?

Reply:
No particular reason, used same conventions that already exist in the code.

Reply:
ok so let's remove the list